<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ObjectDetect</string>
  </property>
  <widget class="QPushButton" name="openCameraButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>210</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OpenCamera</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="openDirectoryButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>250</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OpenDirectory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openVideoButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>290</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OpenVideo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openClassfilerButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>170</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OpenClassifier</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
